hack/lib/scaleup.bash

```diff
@@ -68,6 +68,11 @@ function cluster_scalable {
   if ! oc get machineconfigpool &>/dev/null; then
     return 1
   fi
+  # Prevent scaling for single-node OpenShift.
+  if [[ $(oc get machineconfigpool master -ojsonpath='{.status.machineCount}') == 1 && \
+        $(oc get machineconfigpool worker -ojsonpath='{.status.machineCount}') == 0 ]]; then
+    return 1
+  fi
```

Comment:
There is no way of checking if we are on Single-node Openshift, or Microshift? I mean other than checking if there's only one node?

Reply:
OK. According to https://docs.openshift.com/container-platform/4.13/operators/operator_sdk/osdk-ha-sno.html, there is a way. I will update the PR.

Reply:
If no, maybe some warn log message be nice, instead of that code comment.

Reply:
Pushed.